src/components/ArticlesPage.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ArticlesPage from './ArticlesPage';
+import { MemoryRouter as Router, Route } from "react-router-dom"
+
+let container
```

Comment:
RTL (react-testing-library) will take care of container for us, I think this is redundant.